docs/opera_cli.rst

```diff
@@ -40,32 +40,35 @@ So the simplest way to test ``opera`` is to install it into virtual environment:
-+---------------------+----------------------------------------------+
-| ``opera outputs``   | retrieve outputs from service template       |
-+---------------------+----------------------------------------------+
-| ``opera info``      | show information about the current project   |
-+---------------------+----------------------------------------------+
-| ``opera package``   | retrieve outputs from service template       |
-+---------------------+----------------------------------------------+
-| ``opera unpackage`` | retrieve outputs from service template       |
-+---------------------+----------------------------------------------+
-| ``opera diff``      | compare service templates and instances      |
-+---------------------+----------------------------------------------+
-| ``opera update``    | update/redeploy template and instances       |
-+---------------------+----------------------------------------------+
-| ``opera init``      | initialize the service template or CSAR      |
-+---------------------+----------------------------------------------+
++---------------------+--------------------------------------------------------+
+| CLI command         | Purpose and description                                |
++=====================+========================================================+
+| ``opera deploy``    | deploy TOSCA service template or CSAR                  |
++---------------------+--------------------------------------------------------+
+| ``opera undeploy``  | undeploy TOSCA service template or CSAR                |
++---------------------+--------------------------------------------------------+
+| ``opera validate``  | validate TOSCA service template or CSAR                |
++---------------------+--------------------------------------------------------+
+| ``opera outputs``   | retrieve outputs from service template                 |
++---------------------+--------------------------------------------------------+
+| ``opera info``      | show information about the current project             |
++---------------------+--------------------------------------------------------+
+| ``opera package``   | retrieve outputs from service template                 |
++---------------------+--------------------------------------------------------+
+| ``opera unpackage`` | retrieve outputs from service template                 |
++---------------------+--------------------------------------------------------+
+| ``opera diff``      | compare service templates and instances                |
++---------------------+--------------------------------------------------------+
+| ``opera update``    | update/redeploy template and instances                 |
++---------------------+--------------------------------------------------------+
+|| ``opera notify``   || notify the orchestrator about changes after the       |
+||                    || deployment and run triggers defined in TOSCA policies |
```

Comment:
Does this render correctly? The double pipes seem a bit weird.

Reply:
Yes, the second pipe is for the line block. This is a special syntax for [multi-line table rows](https://stackoverflow.com/questions/13504595/multi-line-table-cell-in-restructuredtext). We already us this here: https://xlab-si.github.io/xopera-opera/opera_cli.html#secrets-and-environment-variables and it renders okay.